docs/source/index.rst

```diff
@@ -394,6 +394,7 @@ TensorFlow and/or Flax.
 
     pretrained_models
     examples
+    troubleshoot
```

Comment:
@sgugger, should I rename it to `troubleshooting` to match `testing`, `contributing`, etc.?

Reply:
Good idea!